tests/test_db_mysql.py

```
@@ -0,0 +1,148 @@
+            result, exec_time = fetch_all("SELECT * FROM test")
+
+            assert result == []
+
+
+class TestDbCommit:
+    """Tests for db_commit function"""
+
+    def test_db_commit_success(self, mock_db_connection):
+        """Test successful commit"""
+        with patch("db_mysql.get_connection", return_value=mock_db_connection):
+            result = db_commit("INSERT INTO test (id, name) VALUES (%s, %s)", [(1, "test")], many=False)
```

Ruff (0.15.12)

[error] Undefined name `db_commit`

(F821)